Sources/PackageGraph/Pubgrub/PartialSolution.swift

```diff
@@ -50,15 +50,15 @@ public final class PartialSolution {
 
     /// Create a new derivation assignment and add it to the partial solution's
     /// list of known assignments.
-    public func derive(_ term: Term, cause: Incompatibility) {
+    mutating public func derive(_ term: Term, cause: Incompatibility) {
```

Comment:
Not sure what the code style for this repo is, but would be good to be consistent `public mutating` vs `mutating public`

Reply:
The [existing `swiftformat` rules](https://github.com/apple/swift-package-manager/blob/main/.swiftformat) should enforce a consistent modifier ordering.